fiftyone/core/state.py

```diff
@@ -75,6 +76,9 @@ def __init__(
         self.spaces = spaces
         self.color_scheme = color_scheme or build_color_scheme()
         self.field_visibility_stage = field_visibility_stage
+        if group_slice is None and view is not None:
+            group_slice = view.group_slice
+        self.group_slice = group_slice
```

Comment:
We want to keep the `group_slice` setting, even when the view, e.g. a patches view, has no group slices